### DISCUSSION TITLE
Spring Cleaning

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#92] [Team Posts]
Hi Guys,

I've done some pretty major updates on articles recently

## Risks Section

https://riskfirst.org/risks/Agency-Risk.   -   A new model of Agency Risk.
https://riskfirst.org/risks/Staging-And-Classifying - A new nice-looking hierarchy of risk types
https://riskfirst.org/risks/Feature-Risk - Updated images and made text more accessible to new-comers
https://riskfirst.org/risks/Communication-Risk - some minor edits
https://riskfirst.org/risks/Complexity-Risk - some minor edits and improved images

## Thinking Section

I've updated nearly all the articles here, updating the language and fixing typos.  

I've also added some nice images to introduce each article, like this one:

![1.  Meet Reality](https://riskfirst.org/images/generated/principles/meet-reality.png)

You can check them out in the child-pages here: 

https://riskfirst.org/thinking/Start

**Comment by BaseMax** [comment]
Hi Rob. good job.
